apps/src/lib/config/genesis.rs

```diff
@@ -523,23 +548,15 @@ pub struct Genesis {
 impl Genesis {
     /// Sort all fields for deterministic encoding
     pub fn init(&mut self) {
-        self.validators.sort();
+        self.validators
+            .sort_by(|val, other| val.account_key.cmp(&other.account_key));
```

Comment:
I'm not sure if this is safe to do as there's no requirement for the account keys to be unique and this order must be deterministic as the genesis hash relies on it. Instead, I think we should add `PartialOrd, Ord, PartialEq, Eq` on `DkgPublicKey` using the encoded data, since we have a wrapper already

Reply:
I'm not sure I understand your recommendation

Reply:
Actually, I can probably revert it back to the old way. The necessary ordering for the DKG is done by ferveo as well